code/modules/mob/living/carbon/life.dm

```diff
@@ -283,6 +283,7 @@
 			setStaminaLoss(health - 2)
 			return
 		setStaminaLoss(max((staminaloss - 2), 0))
+	setBrainLoss(max((brainloss - 1), 0))

```

Comment:
Don't put it INSIDE the Check Staminaloss proc, as that doesn't make sense. Put it into the handle status effects proc, NEXT to the checkstaminaloss proc.